tests/test-cases/vendor-test.yaml

```diff
@@ -11,3 +11,39 @@ tests:
       stderr:
         - "Vendoring is not configured. To set up vendoring, please see https://atmos.tools/core-concepts/vendor/"
       exit_code: 1
+  - name: atmos_vendor_pull
+    enabled: false
```

Comment:
```suggestion
    enabled: true
```

Reply:
In #1097 we disabled OCI tests the right way. Please make sure this doesn't change that. Setting this to `false` means we disabled ALL the tests, not just the OCI.